src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer.java

```diff
@@ -427,10 +427,8 @@ public void start() {
         logger.info("Starting GerritServer: " + name);
 
         //do not try to connect to gerrit unless there is a URL or a hostname in the text fields
-        List<VerdictCategory> categories = config.getCategories();
-        if (categories == null) {
-            categories = new LinkedList<VerdictCategory>();
-        }
+        List<VerdictCategory> categories = config.getCategories() != null ? config.getCategories() : new LinkedList<VerdictCategory>();
```

Comment:
I vote for the previous implementation, it is more readable. It also does only once the `config.getCategories()` call

Reply:
Done